databricks/koalas/tests/test_expanding.py

```diff
@@ -94,14 +94,17 @@ def _test_groupby_expanding_func(self, f):
             repr(getattr(kdf.groupby(kdf.a).expanding(2), f)().sort_index()),
             repr(getattr(pdf.groupby(pdf.a).expanding(2), f)()))
 
-        # TODO: restore below tests when issue #1032 is solved
         # Multiindex column
-        # kdf = ks.DataFrame({'a': [1, 2, 3, 2], 'b': [4.0, 2.0, 3.0, 1.0]})
-        # kdf.columns = pd.MultiIndex.from_tuples([('a', 'x'), ('a', 'y')])
-        # pdf = kdf.to_pandas()
-        # self.assert_eq(
-        #     repr(getattr(kdf.groupby(kdf.a).expanding(2), f)().sort_index()),
-        #     repr(getattr(pdf.groupby(pdf.a).expanding(2), f)()))
+        kdf = ks.DataFrame({'a': [1, 2, 3, 2], 'b': [4.0, 2.0, 3.0, 1.0]})
+        kdf.columns = pd.MultiIndex.from_tuples([('a', 'x'), ('a', 'y')])
+        pdf = kdf.to_pandas()
+        self.assert_eq(
+            repr(getattr(kdf.groupby(kdf.a.x).expanding(2), f)().sort_index()),
```

Comment:
I see the problem. `kdf.a` accessor makes another DataFrame and that makes this operations as an operation on different dataframes.